documentation/tasks/networkdiscovery.md

```diff
@@ -16,21 +16,24 @@ task](networkinventory.html).
 
 This task uses the following protocols to scan IP networks:
 
-* ICMP scan (if nmap is available)
-* NetBIOS scan (if Net::NBName is available, and proper credits provided)
-* SNMP scan (if Net::SNMP is available, and proper credits provided)
+* ICMP scan (if `nmap` is available)
```

Comment:
```suggestion
* ICMP scan (since v2.4.1, if `Net::Ping` is available, try first ICMP Echo and then ICMP TimeStamp)
```
Indeed I introduced a change here in FIA 2.4.1 as nmap license was a problem and now Net::Ping module is used.